src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -267,4 +278,8 @@ private CaseMetadata metadataFromSourceCase(CaseMetadata sourceMetadata) {
     newCaseMetadata.setSecureEstablishment(sourceMetadata.getSecureEstablishment());
     return newCaseMetadata;
   }
+
+  private void addDummyUprnToCase(Case newCase) {
+    newCase.setUprn(String.format("%s%s", dummyUprnPrefix, newCase.getCaseRef()));
```

Comment:
CaseRef is a long, so the formatting string should be `"%s%d"`

Reply:
`%s` is working because it's calling `.toString()` on the caseRef... but that's dodgy